sdk/sqlmanagement/Microsoft.Azure.Management.Sql/src/Generated/ISqlManagementClient.cs

```diff
@@ -662,5 +657,20 @@ public partial interface ISqlManagementClient : System.IDisposable
         /// </summary>
         IServerConnectionPoliciesOperations ServerConnectionPolicies { get; }
 
+        /// <summary>
+        /// Gets the IDistributedAvailabilityGroupsOperations.
+        /// </summary>
+        IDistributedAvailabilityGroupsOperations DistributedAvailabilityGroups { get; }
```

Comment:
I assume these are changes automatically added from other changes in the new API version, right?